omod-1.10/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_10/TestOrderSubclassHandler1_10.java

```diff
@@ -123,6 +124,21 @@ public DelegatingResourceDescription getUpdatableProperties() throws ResourceDoe
 		return orderResource.getUpdatableProperties();
 	}
 	
+	@Override
+	public Model getGETModel(Representation representation) {
+		return null;
+	}
+	
+	@Override
```

Comment:
Were these intentionally left null?

Reply:
Do we need to document handlers? I don't think they appear either as a resource or a property. If needed let me know.

Reply:
Then in that case we should not have these methods in them.

Reply:
However they implements getRepresentationDescription, getCreatableProperties, getCreatableProperties. Is handler considered a resource or a property of some other resource?

Reply:
I would not consider it as a resource. But putting this aside, can you ensure that all other non search handlers that should not just return null, are doing what they should?